client/account/controllers/account.js

```diff
@@ -18,7 +18,7 @@ function AccountController($scope, $sce) {
       $scope.success('Account removed');
       return;
     }
-    $.ajax(`/api/account/${account.provider}/${account.id}`, {
+    $.ajax('/api/account/' + account.provider + '/' + account.id, {
```

Comment:
To fix `npm test`. Lint was breaking and actual tests never ran. Looks like the travis setup is broken, but they run locally for me now.